run_tests

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 DOCKER_IMAGE="libremesh/luatest:1.1"
 if ! docker inspect --type=image $DOCKER_IMAGE > /dev/null 2>&1; then
-    docker build -f Dockerfiles/Dockerfile.unittests -t "$DOCKER_IMAGE" .
+    echo "Pulling docker image '$DOCKER_IMAGE'."
+    echo "If you want to build it locally run 'docker build -f Dockerfiles/Dockerfile.unittests -t $DOCKER_IMAGE .'"
+    docker pull "$DOCKER_IMAGE"
```

Comment:
The thing about using pull instead of build is that if we update this image it will require us to update it manually. Doing the build doesn't require us to update the DOCKER_IMAGE value.
If it is an optimization that was required, we should do it... if not, it might not be that important. Still, your call :) :+1:

Reply:
The image build process does not guarantee that every build makes exactly the same image. The build process tries to make it reproducible but the result may be different. 
So this way we are sure that everyone is using exactly the same images.
And of course the con is that we have to manually do a docker push when we update the image. I have been doing this but noone was using it :P